samples/metaapp/gradle.properties

```diff
@@ -1,2 +1,2 @@
 #Wed Nov 08 14:21:07 CET 2023
-VERSION_CODE=109
+VERSION_CODE=110
```

Comment:
why do we change a version code in the feature PR?

Reply:
because the task to bump the version code failed for the reason explained before and it has to be updated manually